### ISSUE TITLE
Benchmark

### ISSUE THREAD

**Issue by azev77** [UBCECON567/BLPDemand.jl#3]
Hi and thank you for this package. 
Do you think it's worth bench-marking against @jeffgortmaker's [PyBLP](https://github.com/jeffgortmaker/pyblp).py?

**Comment by schrimpf** [comment]
Yes, I think it'd be interesting. It's not a priority for me right now, but I'd welcome any pull requests. Thanks.
